packages/mod.go

```diff
@@ -30,8 +30,7 @@ type Mod struct {
 
 func (m Mod) ExtractPurlsFromManifest() (purls []string) {
 	for _, s := range m.ProjectList.Projects {
-		var version string
-		version = strings.Replace(s.Version, "v", "", -1)
+		version := strings.Replace(s.Version, "v", "", -1)
```

Comment:
Not part of this PR, but it could be nice to use a library for this, https://pkg.go.dev/golang.org/x/mod/semver

Reply:
@greut Im not sure I see  how that library helps here....from what i can tell it will still always return a `v` on the version number. We also use this library as well. https://github.com/Masterminds/semver. That might be an option too but would still requires  recombining the MAJOR, MINOR and PATCH numbers it parsed back into something we want just to get rid of the `v` that may or may not be there.

Reply:
@zendern I see! thanks for the clarification.